common/scala/src/main/scala/org/apache/openwhisk/core/containerpool/ApacheBlockingContainerClient.scala

```diff
@@ -124,13 +129,13 @@ protected class ApacheBlockingContainerClient(hostname: String,
 
           // Negative contentLength means unknown or overflow. We don't want to consume in either case.
           if (contentLength >= 0) {
-            if (contentLength <= maxResponseBytes) {
+            if (contentLength <= maxResponse.toBytes) {
```

Comment:
Do we need `toBytes` here?

Reply:
Yes. Since maxResponse is a ByteSize type, it needs a cast.

```scala
    maxResponse: ByteSize,
```